src/database/migrations/20230605033225_alter_table_users_achievements_hobbies_columns.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return await knex.schema.alterTable("users", (t) => {
+    t.string("achievements", 1000).nullable();
```

Comment:
I would suggest making the column size larger. A limit of 1,000 character seems small for something users could be typing multiple paragraphs into. If you decide to keep it the same I would show a character count next to the text boxes to ensure users don't type to much information and it gets cut off.